include/wx/evtloop.h

```diff
@@ -284,6 +284,10 @@ class WXDLLIMPEXP_BASE wxEventLoopManual : public wxEventLoopBase
     #include "wx/msw/evtloopconsole.h"
 #endif
 
+#if defined(__WXQT__)
```

Comment:
Is it normal that we include _both_ `wx/unix/evtloop.h` or `wx/msw/evtloopconsole.h` _and_ `wx/qt/evtloop.h`?

Because either `__UNIX__` or `__WINDOWS__` is still going to be defined, even when `__WXQT__` also is, isn't it?

Besides, the console event loop is really supposed to be tookit-independent, so that you could use the same base library with either GTK or Qt-based core, for example. It would be nice if we could keep it like this.

Reply:
It is probably not correct, but I tried the get a successful build by changing as little `WXQT`-code as possible.

Yes, in my case `__WINDOWS__ ` is still defined and the Windows `wxConsoleEventLoop` is compiled as well. That is why I had to rename the `WXQT` console event loop to `wxQtConsoleEventLoop`.

Following your paradigm, `WXQT` should not define a console event loop? And all `WXQT` code should be part of `core`?